### PR TITLE
more: add Spanish translation

### PR DESCRIPTION
**Pull request by AlejandroCruz-0**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/common/more.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Abre un archivo:
+
+`more {{archivo}}`
+
+- Página abajo:
+
+`<Espacio>`
+
+- Busca una termino (La tecla `n` se mueve al siguiente resultado):
```
Comment (navarroaxel) [inline pages.es/common/more.md:13]: La mejor traducción sería cadena (de caracteres), ya que termino hace más referencia a busqueda por keywords o más inteligente:
```suggestion
- Busca una cadena (presione `n` para ir a la siguiente coincidencia):
```

File: pages.es/common/more.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`/{{termino de busqueda}}`
```
Comment (navarroaxel) [inline pages.es/common/more.md:15]: We use variables in `snake_case`, if you use `cadena` in the example description, I think this should be:
```suggestion
`/{{cadena}}`
```

File: pages.es/common/more.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# more
+
+> Abre un archivo para lectura interactiva, permitiendo desplazar y buscar (Solamente en una dirección, adelante).
+
+- Abre un archivo:
+
+`more {{archivo}}`
```
Comment (navarroaxel) [inline pages.es/common/more.md:7]: We use `path/to/file` now:
```suggestion
`more {{ruta/al/archivo}}`
```

File: pages.es/common/more.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# more
+
+> Abre un archivo para lectura interactiva, permitiendo desplazar y buscar (Solamente en una dirección, adelante).
```
Comment (navarroaxel) [inline pages.es/common/more.md:3]: 1. `more` accepts scroll backwards in macOS and Linux now.
2. `more` is an old page. You can add the more info link to the English page too.
3. I think `navegar` sounds better.
```suggestion
> Abre un archivo para lectura interactiva, permitiendo navegar y buscar.
> Más información: <https://manned.org/more>.
```

File: pages.es/common/more.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+- Abre un archivo:
+
+`more {{archivo}}`
+
+- Página abajo:
```
Comment (navarroaxel) [inline pages.es/common/more.md:9]: I think we say be a little more verbose:
```suggestion
- Avanza hacia la siguiente página:
```

**Review by navarroaxel — COMMENTED** [review]
Hey @AlejandroCruz-0 welcome and thanks for your submission. We need more Spanish translations!

I left some comments.

**Comment by AlejandroCruz-0** [comment]
ok i'll make the suggested changes, all of the suggested changes seem sensible. Thanks for the feedback.

File: pages.es/common/more.md (hunk 5 above)
Comment (navarroaxel) [reply pages.es/common/more.md:10]: @AlejandroCruz-0 what about this one?

Comment (AlejandroCruz-0) [reply pages.es/common/more.md:10]: oh wow, i forgot about that one, oops. I'll fix it.

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]
Thank you @AlejandroCruz-0!

Welcome to tldr-pages! 🚀

**Pull request merged**